examples/core-ping.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "http://mozilla.org/schemas/glean/ping",
```

Comment:
This is sent _in_ the payload? What's this for?

Reply:
This matches it to the schema -- see the `$id` property in `ping.schema.json`.  But from discussion on Slack, I see that `docType` is used elsewhere in telemetry.  No problem with doing that instead of or in addition to this.

Reply:
Going with `docType == glean`.